lib/dealership.rb

```diff
@@ -0,0 +1,86 @@
+class Dealership
+  attr_reader :dealer_name,
+              :address,
+              :inventory,
+              :inventory_count,
+              :value
+
+  def initialize(dealer_name, address)
+    @dealer_name      = dealer_name
+    @address          = address
+    @inventory        = []
+    @inventory_count  = 0
+    @value            = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    if @inventory_count = 0
+      return false
+    else
+      return true
+    end
+  end
```

Comment:
You are doing a variable assignment here rather than checking if it is equal

```suggestion
  def has_inventory?
    if @inventory_count == 0
      return false
    else
      return true
    end
  end
```